### PR TITLE
Update Norse case test to use JuMP.is_solved_and_feasible

### LINKED ISSUE #593: Change test for the Norse case study
### What and Why

We test the Norse and check the objective function in the tests. However, that case study is continuously evolving and changing, so we really don't care about the optimal value. What we do care about is that it is feasible and optimal. Therefore, we should change the test to use `is_solved_and_feasible` function from JuMP. By doing this, we avoid updating an objective function that is not meaningful for our tests.

### Possible Drawbacks

_No response_

### Related Issues

_No response_

### PR DESCRIPTION
**Pull request by datejada**
# Pull request details

## Describe the changes made in this pull request

Using `JuMP.is_solved_and_feasible` makes us focus on the feasibility and optimality of the Norse case rather than the objective value.

## List of related issues or pull requests

Closes #593 

## Collaboration confirmation

As a contributor I confirm

-   [x] I read and followed the instructions in README.dev.md
-   [NA] The documentation is up to date with the changes introduced in this Pull Request (or NA)
-   [x] Tests are passing
-   [x] Lint is passing

### CODE REVIEW

File: test/test-case-studies.jl [hunk 1]
```diff
@@ -9,7 +9,7 @@
     end
     for (optimizer, parameteres) in parameters_dict
         energy_problem = run_scenario(dir; optimizer = optimizer, parameters = parameteres)
-        @test energy_problem.objective_value ≈ 1.9848306581e8 rtol = 1e-8
+        @test JuMP.is_solved_and_feasible(energy_problem.model) == true
```
Comment (abelsiqueira) [inline test/test-case-studies.jl:12]:
```suggestion
        @test JuMP.is_solved_and_feasible(energy_problem.model)
```

**Review by abelsiqueira — CHANGES_REQUESTED** [review]
Small stuff

**Comment by datejada** [comment]
> Small stuff

Great! thanks, I wasn't sure if that `== true` was needed. So, thanks! 😄

**Review by abelsiqueira — APPROVED** [review]
Thanks!

**Pull request merged**